packages/bindings/binding.gyp

```diff
@@ -4,9 +4,9 @@
     'sources': [
       'src/serialport.cpp'
     ],
-    'include_dirs': [
-      '<!(node -e "require(\'nan\')")'
-    ],
+    'include_dirs': ["<!(node -p \"require('node-addon-api').include_dir\")"],
```

Comment:
I've always seen this setting as:

```gyp
'include_dirs': ["<!@(node -p \"require('node-addon-api').include_dir\")"],
```

Specifically, the `@` seems to be missing.

Note, `dependencies` generally doesn't use the `@`.

Reply:
This seems to have been revised as part of https://github.com/nodejs/node-addon-api/commit/f27623ff611645cd7d927f0de81158a93e960542#diff-55ad3a91265d731555313fdee9af5e109244b0e825c4aff0b7cce3c78b07be97